Index: meego-lsb-4.0/lsb-release-2.0/lsb_release
===================================================================
--- meego-lsb-4.0.orig/lsb-release-2.0/lsb_release
+++ meego-lsb-4.0/lsb-release-2.0/lsb_release
@@ -188,15 +188,6 @@ EnhancedGetopt() {
 
 # Get/Init LSB infos (maybe Distrib infos too)
 GetLSBInfo() {
-    if [ -f "$INFO_ROOT/$INFO_LSB_FILE" ]
-    then
-        # should init at least LSB_VERSION
-        . "$INFO_ROOT/$INFO_LSB_FILE"
-    fi
-    if [ -z "$LSB_VERSION" ]
-    then
-	LSB_VERSION=$MSG_NA
-    else
 	# if we found LSB_VERSION, continue to look in directory
 	if [ -d "$INFO_ROOT/$INFO_LSB_DIR" ]
 	then
@@ -205,7 +196,6 @@ GetLSBInfo() {
 		LSB_VERSION=$LSB_VERSION:`basename $tag`
 	    done
 	fi
-    fi
 }
 
 # Get the whole distrib information string (from ARG $1 file)
